drivers/hmis/app/models/hmis/role.rb

```diff
@@ -163,13 +163,70 @@ def self.permissions_with_descriptions
         sub_category: 'Management',
       },
       can_manage_units: {
-        description: 'Ability to manage bed and unit capacity in the project',
+        description: 'Ability to manage bed and unit capacity in the project. If Coordinated Entry is enabled in the project, users can also mark units as available for receiving referrals.',
         administrative: false,
         access: [:editable],
         category: 'Project Access',
         sub_category: 'Management',
       },
-      can_manage_incoming_referrals: {
+      can_view_prioritized_client_lists: {
+        description: 'Ability to view client waitlists in the project',
+        administrative: false,
+        access: [:viewable],
+        category: 'Project Access',
+        sub_category: 'Referrals',
+      },
+      can_start_referrals: {
+        description: 'Ability to initiate referrals from the client waitlist in the project',
+        administrative: false,
+        access: [:editable],
+        category: 'Project Access',
+        sub_category: 'Referrals',
+      },
+      can_view_referrals: {
```

Comment:
To discuss
- the addition of these permissions may cause confusion for existing customers, particularly those using existing referral functionality. discuss if there is an easy way to implement https://github.com/open-path/Green-River/issues/7703 aka hiding these in the UI if the feature flag is off

Reply:
Decision from standup:
- Minimal approach that would hide these new permissions, a la https://github.com/open-path/Green-River/issues/7703#issuecomment-2891267555
- Add todos for future work around feature

Reply:
Thanks @gigxz  for the sketch of an approach in your linked comment. I've implemented it as you suggested and it looks like it works well enough for now.

I haven't done anything in `RolesController`, since my understanding from our standup discussion was that this is sort of a stopgap that we don't necessarily plan to continue supporting. Let me know if you don't agree!

Reply:
Discussed - sufficient for now

Reply:
Yep - updated proc change looks good!